trpcultivate_germplasm/trpcultivate_germplasm.install

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * @file
+ * Provides functionality required for installation of the module.
+ */
+
+/**
+ * Implement hook_install()
+ */
+function trpcultivate_germplasm_install(){
+
+  // Now create the stock_synonym linking table.
+  $table = 'stock_synonym';
+  $schema = [
+    'table' => 'stock_synonym',
+    'description' => 'Linking table between stock and synonym.',
+    'fields' => [
+      'stock_synonym_id' => [
+        'type' => 'serial',
+        'not null' => TRUE,
+      ],
+      'synonym_id' => [
+        'size' => 'big',
+        'type' => 'int',
+        'not null' => TRUE,
+      ],
+      'stock_id' => [
+        'size' => 'big',
+        'type' => 'int',
+        'not null' => TRUE,
+      ],
+      'pub_id' => [
+        'size' => 'big',
+        'type' => 'int',
+        'not null' => TRUE,
+      ],
+      'is_current' => [
+        'type' => 'int',
+        'default' => 0,
+      ],
+      'is_internal' => [
+        'type' => 'int',
+        'default' => 0,
+      ],
+    ],
+    'primary key' => [
+      'stock_synonym_id',
+    ],
+    'indexes' => [
```

Comment:
There needs to be a unique key for synonym_id + stock_id + pub_id added here. Don't forget to add the same in the test trait.
```suggestion
   'unique keys' => [
      'stock_synonym_c1' => ['synonym_id', 'stock_id', 'pub_id'],
    ],
    'indexes' => [
```

Reply:
Addressed in commit https://github.com/TripalCultivate/TripalCultivate-Germplasm/pull/6/commits/6c363dacdc437d980191d6e09612d709fbdc33f7!